### PR TITLE
Update timers extension

### PR DESCRIPTION
**Pull request by iamharbie**
## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Hi @iamharbie 👋 

What did you change? Please describe in the PR and in a CHANGELIOG entry 😉

File: extensions/timers/package.json [hunk 1]
```diff
@@ -108,21 +89,13 @@
       "title": "Timers Menu Bar",
       "description": "View, start, or stop existing countdown timers from the Menu Bar.",
       "mode": "menu-bar",
-      "interval": "10s"
+      "interval": "1s"
```
Comment (ThatNerdSquared) [inline extensions/timers/package.json:92]: Maybe an unfinished PR, accidentally opened?

From the [Raycast docs](https://developers.raycast.com/information/lifecycle/background-refresh#manifest):

> The minimum value is 10 seconds (10s)

**Review by lokot0k — COMMENTED** [review]
I quite don't get it... So the one substantial change is redefinition of app's execution interval? Why then the rest of the fields ended up moved? Or I got it wrong?

**Comment by pernielsentikaer** [comment]
Closing, feel free to request a reopen if needed 😊

**Pull request closed without merging**